commons-dlp/src/main/java/org/exoplatform/commons/dlp/connector/DlpServiceConnector.java

```diff
@@ -76,4 +76,6 @@ public void setEnable(boolean enable) {
   public abstract void removePositiveItem(String itemReference);
 
   public abstract String getItemUrl(String itemReference);
+
+  public abstract boolean checkExternal(String userId);
```

Comment:
I think we should implement the method here and not in the children class.
In addition, do we already have a utils function which compute that ?